haystack/components/rankers/diversity.py

```diff
@@ -0,0 +1,238 @@
+import logging
+from typing import Any, Dict, List, Literal, Optional
+
+from haystack import ComponentError, Document, component, default_from_dict, default_to_dict
+from haystack.lazy_imports import LazyImport
+from haystack.utils import ComponentDevice, Secret, deserialize_secrets_inplace
+
+logger = logging.getLogger(__name__)
+
+
+with LazyImport(message="Run 'pip install \"sentence-transformers>=2.2.0\"'") as torch_and_transformers_import:
```

Comment:
This is a small nit pick but could we rename `torch_and_transformers_import` to `torch_and_sentence_transformers_import` to reflect the two main libraries we are importing?

Reply:
Yes, I will update that.